scripts/build_gh_pages

```diff
@@ -4,71 +4,70 @@ readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
 # shellcheck source=scripts/common
 source "$SCRIPTS_DIR/common"
 
-readonly SRC_SUBDIRS=(sdk/rust oak/server/rust)
-readonly DOC_SUBDIRS=(sdk server)
-
 # Update the gh-pages branch. Note that `cargo doc` is **not deterministic** so
```

Comment:
Do you find that `cargo doc` works deterministically for you?  (I'm guessing not, given a510f9189c880630bf7e7f88b74a932436d5e55a)

If not, aren't we still going to need something like the hacky change-detection stuff?

Reply:
Wrong commit, that should have been 146c3c5f14faa09c0e2c6a7dab881cfe1f65fb4a that only seems to show indexing changes

Reply:
It seems deterministic on my machine, so I'm removing the logic and the comment, but let me know if you find out that's not the case.